pypeclub/OpenPype
# Hiero: add support for task tags

## Brief description
Task tags in Hiero are supported now

## Additional notes
Since we needed to fix process for the case where user could collect deactivated instances and then decide to activate some and publish, I have to implement some changes to `integrate ftrack hierarchy` and `extract avalon hierarchy`. Now it is possible to activate instances and then all relevant hierarchy will be also published. 

## Testing notes:
1. open testing hiero project
2. drop task tags from tags panel to your track items which are going to be published
3. publish > you can only allow shot instances
4. ftrack should have your task collection on each shot

## dependency
3.10.x alternative https://github.com/pypeclub/OpenPype/pull/3279
